### PR TITLE
Closes #2717 News - Telemetry

### PR DESCRIPTION
**Pull request by cnevinc**
See:
https://github.com/mozilla-tw/FirefoxLite/issues/3173#issuecomment-470475293
https://github.com/mozilla-tw/FirefoxLite/issues/2712#issuecomment-470482009

### CODE REVIEW

File: app/src/main/java/org/mozilla/rocket/content/ContentPortalView.kt [hunk 1]
```diff
@@ -97,10 +98,11 @@ class ContentPortalView : CoordinatorLayout, ContentAdapter.ContentPanelListener
         if (visibility == VISIBLE) {
             return
         }
-
         HomeFragmentViewState.lastOpenNews()
         contentPortalListener?.onShow()
 
+        TelemetryWrapper.openLifeFeedNews()
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/rocket/content/ContentPortalView.kt]: It's actually opening the content portal, consider renaming this with Eddie.

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt [hunk 2]
```diff
@@ -1269,6 +1277,39 @@ object TelemetryWrapper {
...
+            category = Category.ACTION,
+            method = Method.CLICK,
+            `object` = Object.PANEL,
+            value = Value.NEWS,
+            extras = [TelemetryExtra(name = Extra.POSITION, value = "1,2,3..."),
+                    TelemetryExtra(name = Extra.FEED, value = "Newspoint,DainikBhaskar.com"),
+                    TelemetryExtra(name = Extra.SOURCE, value = "India TV,Business World,HW News English...etc"),
+                    TelemetryExtra(name = Extra.CATEGORY, value = "Uttar Pradesh,National,Tech Knowledge....etc"),
+                    TelemetryExtra(name = Extra.SUB_CATEGORY, value = "top-news,entertainment,Lucknow...etc")])
+    fun clickOnNesItem(pos: String, feed: String, source: String, category: String, subCategory: String) {
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: typo in function name

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt [hunk 3]
```diff
@@ -1963,11 +2004,14 @@ object TelemetryWrapper {
 
         init {
             lazyInit()
+            Log.d(TAG, "EVENT:$category/$method/$`object`/$value")
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: Not sure if intended to keep these log.

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt [hunk 4]
```diff
@@ -1963,11 +2004,14 @@ object TelemetryWrapper {
...
+
             telemetryEvent = TelemetryEvent.create(category, method, `object`, value)
             firebaseEvent = FirebaseEvent.create(category, method, `object`, value)
         }
 
         fun extra(key: String, value: String): EventBuilder {
+            Log.d(TAG, "EXTRA:$key/$value")
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: ditto

Comment (cnevinc) [reply app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: yes

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt (hunk 3 above)
Comment (cnevinc) [reply app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: yes. So developers can debug without setting the break point.

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt (hunk 2 above)
Comment (cnevinc) [reply app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt]: fixed

File: app/src/main/java/org/mozilla/rocket/content/ContentPortalView.kt (hunk 1 above)
Comment (cnevinc) [reply app/src/main/java/org/mozilla/rocket/content/ContentPortalView.kt]: fixed.

**Comment by cnevinc** [comment]
per discussion with @elin-moco 
It's okay if we want to clearly separate UI action from User intention when we send the event.
I'll keep the original design and add generalized UI pings if needed in the future.

**Pull request merged**